econml/inference.py

```diff
@@ -1134,8 +1152,7 @@ def stderr_point(self):
         """
         return np.sqrt(self.stderr_mean**2 + self.std_point**2)
 
-    @property
-    def conf_int_point(self):
+    def conf_int_point(self, *, alpha=None, tol=None):
         """
         Get the confidence interval of the point estimate of each treatment on each outcome for sample X.
 
```

Comment:
Add parameter documentation for alpha, tol

Reply:
Done